setup.py

```diff
@@ -37,11 +37,11 @@
         'numpy>=1.8.2',
         'sgp4>=2.3',
         'requests',
+        "scipy>=0.16",
```

Comment:
We had `scipy` in `extras_require` to avoid having install it in "constrained or low resources platforms" (nudge nudge wink wink). I think it would be nice if we could keep it like that, for example by conditionally importing `scipy` and giving a warning to the user if it's not available, while not breaking the rest of the code. What do you think?

Reply:
I think it is a great idea. So, besides handling the import exception, scipy should be removed from `requirmentes.txt` and put back into the `extra_require` section of the `setup.py `?

Reply:
Yes, let's remove it from `requirements.txt` and put it back in `extras_require`. We will also have to change this:

https://github.com/satellogic/orbit-predictor/blob/ea1d727bb01a5ee30d83a85c9cc31f9959355e4d/.travis.yml#L13

to `pip install ... .[dev,fast]`.

Reply:
How should the tests of this new feature should be handled to avoid an exception if there is no scipy in the dev environment?